notebooks/community/bigquery_ml/bqml-online-prediction.ipynb

```diff
@@ -0,0 +1,1272 @@
+{
```

Comment:
<p>Add title.</p><p>Give some explanation on why helper function</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/736/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
Added title:
`#### Helper function for running BigQuery queries`

Elaborated on explanation:
`Use a helper function for sending queries to BigQuery, including providing error messages from BigQuery if there is an issue with the input `sql` query.`